src/client/cockpit-client-ssh

```diff
@@ -52,14 +52,20 @@ def cockpit_client_ssh():
     askpass = os.path.join(xdg_data_home, 'cockpit-client-askpass')
     shutil.copy(__file__, askpass, follow_symlinks=False)
 
-    host = sys.argv[1]
+    connect_to = sys.argv[0]
```

Comment:
This is wrong -- argv[0] is the program name. argv[1] was right.

Reply:
oops.